src/replication.c

```diff
@@ -388,8 +388,7 @@ void feedReplicationBuffer(char *s, size_t len) {
     /* Try to trim replication backlog since replication backlog may exceed
      * our setting when we add replication stream. Note that it is important to
      * try to trim at least one node since in the common case this is where one
-     * new backlog node is added and one should be removed. see also
-     * freeMemoryGetNotCountedMemory. */
```

Comment:
why did you remove the reference to the eviction code? i think it may be useful

Reply:
when i reviewed code, i thought it is not much related with `freeMemoryGetNotCountedMemory `, i forgot why i added it, you think they are related?

Reply:
I think i was trying to say that this is related to the feedback loop concern when DEL commands are added to the backlog during eviction.
i was probably trying refer to the big comments we added there.

maybe it should be even be: `see comments in freeMemoryGetNotCountedMemory for details`
i.e. not to refer to the function, but rather to the comments there explaining what would happen when the backlog grows.